example_config/EUR-Lex-v2/bigru.yml

```diff
@@ -0,0 +1,32 @@
+# data
+data_dir: data/EUR-Lex-v2
```

Comment:
-v2 is our own name.. I think they call it Eur-Lex-57k. Also that's the name we used in libsvm data.